app/src/main/res/drawable/onboarding_padded_background.xml

```diff
@@ -2,7 +2,7 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="@color/onboarding_padded_background_color">
+    android:color="#1F000000">
```

Comment:
I see this color also exists as `shadow_normal_theme`. Maybe it can be migrated to that?

Reply:
`@color/onboarding_padded_background_color` was a static `#1F000000` (no difference between dark/light theme), which is why I chose to just inline directly as well. Unfortunately, `shadow_normal_theme` has both a dark and light theme variation, so it cannot be used in this situation.